gnuradio-runtime/lib/qa_logger.cc

```diff
@@ -47,6 +45,6 @@ qa_logger::t1()
   GR_LOG_WARN(LOG,"test from c++ WARN");
   GR_LOG_ERROR(LOG,"test from c++ ERROR");
   GR_LOG_FATAL(LOG,"test from c++ FATAL");
-  CPPUNIT_ASSERT(true);
+  BOOST_CHECK(true);
```

Comment:
I … uh. What?

Reply:
If this is a setup function, we shouldn't be defining it as testcase. If this is a test case, we shouldn't be checking `true`.

Reply:
Yeah I know. But I'm not changing the tests *and* the unit test framework in the same commit. I found a bunch of these cringeworthy things :grimacing:

Reply:
:grin:  good approach.